crates/matrix-sdk-ui/src/room_list_service/sorters/recency.rs

```diff
@@ -66,11 +64,11 @@ where
 }
 
 /// Create a new sorter that will sort two [`Room`] by recency, i.e. by
-/// comparing their [`matrix_sdk_base::RoomInfo::recency_timestamp`] value. The
-/// `Room` with the newest recency timestamp comes first, i.e. newest < oldest.
+/// comparing their [`matrix_sdk_base::RoomInfo::recency_stamp`] value. The
```

Comment:
An import behind a `#[cfg(doc)]` is usually much nicer.

```suggestion
/// comparing their [`RoomInfo::recency_stamp`] value. The
```

Reply:
If you prefer a shorter link, I prefer to use the following format:

```md
[`RoomInfo::recency_stamp`] blah blah blah

[`RoomInfo::recency_stamp`]: matrix_sdk_base::RoomInfo::recency_stamp
```

Reply:
I think that this format won't autodetect dead links. So leave it as is if you disagree with the import and `#[cfg(doc)]`.